src/gui/PinchZoom.ts

```diff
@@ -52,6 +53,12 @@ export class PinchZoom {
 	// null if there was no previous touch position related to dragging
 	private lastDragTouchPosition: CoordinatePair | null = null
 
+	// Apple considers that a press event is a long press after 500ms
+	// Google considers after 400ms
+	// https://developer.apple.com/documentation/uikit/uilongpressgesturerecognizer/1616423-minimumpressduration
+	// https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/ViewConfiguration.java
+	private LONG_PRESS_MIN_MS = 400
```

Comment:
```suggestion
	private readonly LONG_PRESS_MIN_MS = 400
```
This could be made `readonly`.  Also I wonder if we could switch between 400 & 500 depending whether the client is on Android or iOS.